examples/test-animate-color/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Animate Color</title>
+    <meta name="description" content="Animate Color - A-Frame">
+    <script src="../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene stats="true">
+      <a-sky color="#afafaf"></a-sky>
+      <a-box color="white" position="2 1 -10">
```

Comment:
@ngokevin I used setTimeouts in that to stagger the animation emission of events.  If you guys are ok with that.  Some repo's are real against timeouts so I wasn't sure if I should use it as an example.